app/controllers/api/v1/submissions_controller.rb

```diff
@@ -1,12 +1,20 @@
 class Api::V1::SubmissionsController < ApplicationController
   def create
     submission = Submission.create!(submission_params)
-    render json: SubmissionsSerializer.new(submission), status: 201
+    attach_submission_admin(submission)
+
+    json_response(SubmissionsSerializer.new(submission), :created)
   end
 
   private
 
   def submission_params
     params.require(:submission).permit(:email, :profile, :video, :genre, :name, :song_title, :winner)
   end
+
+  def attach_submission_admin(submission)
+    Admin.all.each do |admin|
+      SubmissionAdmin.create!(admin_id: admin.id, submission_id: submission.id)
+    end
+  end
```

Comment:
I would consider this data logic and recommend that you move it to the Submission model.

Reply:
Ok sounds good, I will move the logic and re-push the change.

Reply:
@BrianZanti I have updated this logic and moved it to the model as a after_create callback action. I have also added testing for the new method. Please let me know if any additional changes are needed. Thanks!